beacon-chain/core/state/state_test.go

```diff
@@ -1,4 +1,4 @@
-package state
+package state_test
```

Comment:
Made this into its own package to prevent import cycles and blackbox some of these state transition tests.